src/tests/Microsoft.VisualStudio.Composition.Tests/RuntimeCompositionTests.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace Microsoft.VisualStudio.Composition.Tests
+{
+    using System;
+    using Xunit;
+
+    public class RuntimeCompositionTests
+    {
+        [Fact]
+        public void TestEmptyCatalogTest()
+        {
+            var configuration = CompositionConfiguration.Create(TestUtilities.EmptyCatalog);
+            var composition = RuntimeComposition.CreateRuntimeComposition(configuration);
+            var exports = composition.GetExports(typeof(IDisposable).ToString());
+            Assert.Equal(0, exports.Count);
```

Comment:
nit: `Assert.Empty(exports);` can result in a more useful error message when the test fails. Not a big deal, but something to consider next time.